assets/openobserve/ingest.py

```
@@ -0,0 +1,53 @@
+        data = []
+
+        for line in file:
+            linee = line.strip()
+            if not linee:
+                continue
+            #a = base32_encode_json(json.loads(linee))
+            #data.append(a)
+            data.append(json.loads(linee))
+
+            if len(data)>50000:
+                res = requests.post(openobserve_url, headers=headers, data=json.dumps(data))
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)